languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -47925,6 +47925,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="La hausse"><marker>L'hausse</marker> des profits pétroliers.</example>
             </rule>
             <rule>
+                <antipattern><!-- see languagetool-premium/issues/5077 -->
+                    <token regexp="yes">s'</token>
+                    <token regexp="yes">te</token>
```

Comment:
Nitpick (non-blocking): Maven test will complain with a warning because of the regexp="yes"